src/common/components/MktSideRecommendations.jsx

```diff
@@ -207,13 +206,13 @@ function MktSideRecommendations({ title, endpoint, technologies, containerPaddin
               const tags = [];
 
               return (
-                <Container border="1px solid" borderColor={{ base: 'default', md: 'success' }} key={recom?.slug} course={recom} courses={recommendations} borderRadius={rest.borderRadius} padding={containerPadding}>
+                <Container border="1px solid" borderColor={{ base: 'default', md: 'blue.400' }} key={recom?.slug} course={recom} courses={recommendations} borderRadius={rest.borderRadius} padding={containerPadding}>
```

Comment:
Instead of adding a default blue color, that color is coming from the course, you should use the color that comes from the backend if you receive it, otherwise show green color